src/i18n/strings/en_EN.json

```diff
@@ -804,9 +804,9 @@
     "No recent messages by %(user)s found": "No recent messages by %(user)s found",
     "Try scrolling up in the timeline to see if there are any earlier ones.": "Try scrolling up in the timeline to see if there are any earlier ones.",
     "Remove recent messages by %(user)s": "Remove recent messages by %(user)s",
-    "You are about to remove %(count)s messages by %(user)s. This cannot be undone. Do you wish to continue?|other": "You are about to remove %(count)s messages by %(user)s. This cannot be undone. Do you wish to continue?",
+    "You are about to remove %(amount)s message(s) by %(user)s. This cannot be undone. Do you wish to continue?": "You are about to remove %(amount)s message(s) by %(user)s. This cannot be undone. Do you wish to continue?",
     "For a large amount of messages, this might take some time. Please don't refresh your client in the meantime.": "For a large amount of messages, this might take some time. Please don't refresh your client in the meantime.",
-    "Remove %(count)s messages|other": "Remove %(count)s messages",
```

Comment:
why are we moving away from proper pluralisation?

Reply:
because I couldn't be bothered figure out how it works I suppose. Is there a good example somewhere?

Reply:
like it was I believe, the word `count` is special, it dictates whether to use `one` or `two` or `three` or `other` etc

Reply:
I see. I was having problems with the translation when using `count`, so was using it without knowing. The translation would be empty with `count=1`. How would I fix this, by adding a `...|one` string in `en_en.json`?

Reply:
Yup, that should be it